docs/source/conf.py

```diff
@@ -16,6 +16,28 @@
 import os
 import shlex
 
+# use Breathe to import Doxygen XML output into our Sphinx documentation
+root_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), "../.."))
+source_directories = [
+    "packages/Utils/src",
+    "packages/Interface/src",
```

Comment:
We don't actually to add it all.  It is only needed if we reference some class that is defined in these directories.  Here for instance `packages/Utils/src` is not actually needed since I only added doc for the interface.  Does that make sense?